### ISSUE TITLE
Question: Are we able to set exec_properties to a set of actions with specific mnemonic ?

### ISSUE THREAD

**Issue by st0rm23** [bazelbuild/bazel#20039]
Hi, 
  
  May I ask a question that whether bazel can add exec_properties to a specific mnemonic ? My goal is to assign all CppLink actions to a specific remote workers. 
  It looks --modify_execution_info can add tags to specific mnemonic, but it is not able to add exec_properties. And the platform configuration is not able to limit the exec_properties to actions with specific mnemonics.

  So I just come here to find some help, thanks!

**Comment by rakshitasingh05** [comment]
@st0rm23 Could you please elaborate your issue in details and provide the complete steps to reproduce this issue?

**Comment by st0rm23** [comment]
> elaborate

Sorry for the misleading. What I ask is about the bazel usage, not a bug. So there's nothing to reproduce. But I can try to give more details. 

For example, we can apply a customized exec_properties during a build like `bazel build <target> --remote_default_exec_properties=use_large_worker=true --remote_executor=...`. The problem is that it sets the exec_properties to all actions in <target>. I just hope to attach exec_properties=use_large_worker=true to CppLink actions other than CppCompile actions.

If I use `bazel build <target> --modify_execution_info=CppLink:+use_large_worker=true --remote_executor=...` It does not work, and it looks --modify_execution_info only supports adding tags other than key-value pairs. 

So I just ask here if we have a way to attach exec_properties to specific mnemonic like CppLink.

**Comment by tjgq** [comment]
I don't believe we have a way to do this, sorry. I think it's a reasonable feature request; we might also want to rethink the distinction between execution properties and execution info more broadly (see https://groups.google.com/g/bazel-discuss/c/CEw6epfmbR8/m/hw04DWGcDAAJ for relevant prior discussion).

**Comment by linbinghui** [comment]
> I don't believe we have a way to do this, sorry. I think it's a reasonable feature request; we might also want to rethink the distinction between execution properties and execution info more broadly (see https://groups.google.com/g/bazel-discuss/c/CEw6epfmbR8/m/hw04DWGcDAAJ for relevant prior discussion).

I see, thanks for replying. Really hope there would be a new feature to support this!

**Comment by st0rm23** [comment]
> I don't believe we have a way to do this, sorry. I think it's a reasonable feature request; we might also want to rethink the distinction between execution properties and execution info more broadly (see https://groups.google.com/g/bazel-discuss/c/CEw6epfmbR8/m/hw04DWGcDAAJ for relevant prior discussion).

I see, thanks for replying. Really hope there would be a new feature to support this!

**Comment by CaerusKaru** [comment]
Still needed for improved RBE experience.
